src/uct/ib/dc/dc_mlx5.c

```diff
@@ -1186,10 +1202,12 @@ ucs_status_t uct_dc_mlx5_iface_fc_handler(uct_rc_iface_t *rc_iface, unsigned qp_
         status = uct_dc_mlx5_iface_fc_grant(&dc_req->super.super);
         if (status == UCS_ERR_NO_RESOURCE){
             uct_dc_mlx5_ep_do_pending_fc(ep, dc_req);
-        } else {
-            ucs_assertv_always(status == UCS_OK,
-                               "Failed to send FC grant msg: %s",
-                               ucs_status_string(status));
+            status = UCS_OK;
+        } else if (status != UCS_OK) {
+            ucs_diag("fc_ep %p: failed to send %s: %s", ep,
+                     uct_dc_mlx5_iface_fc_pure_grant_str(
+                             dc_req, buf, ucs_static_array_size(buf)),
```

Comment:
sizeof(buf) for consistency with other code

Reply:
done